### ISSUE TITLE
React-Native

### ISSUE THREAD

**Issue by ugar0ff** [bl00mber/react-phone-input-2#382]
Do you plan to support React-Native? 
Now, when I use PhoneInput I get error.
```
import PhoneInput from 'react-phone-input-2';
<PhoneInput />
```
_ReactNativeJS: Invariant Violation: View config getter callback for component `div` must be a function (received `undefined`)._

**Comment by AdamMomen** [comment]
I agree @ugar0ff, this package is highly customizable, it would be a great idea to make work on react-native as well.

**Comment by Davete0302** [comment]
Would be nice if this supports react native
